### DISCUSSION TITLE
how to uninstall?

### DISCUSSION THREAD

**Discussion by FrontoOptimo** [storaged-project/udisks#1141] [Q&A] [answered]
new to linux and i cant uninstall udisk? is there a line code for it?

**Comment by vojtechtrefny** [comment, marked as answer]
Uninstallation steps depend on your distribution and/or package manager/installation method you use. For example in fedora you can use `sudo dnf remove udisks2`, in Ubuntu/Debian `sudo apt remove udisks2`. Please consult your distribution documentation or ask on a distribution specific help channel.
